backend/src/models/AffiliationManager.js

```diff
@@ -0,0 +1,31 @@
+const AbstractManager = require("./AbstractManager");
+
+class AffiliationManager extends AbstractManager {
+  static table = "affiliation";
+
+  findProducts(search) {
+    return this.connection.query(
+      `select * from product inner join ${this.table} on product.id = product_id inner join keyword on keyword.id = keyword_id where keyword_name like '%${search}%'`
```

Comment:
see the feedbacks in this PR : https://github.com/WildCodeSchool/2022-03-JS-Reims-project-3-shop-around-me/pull/10